appendix/backup-and-restore/migrate-hosts.rst

```diff
@@ -81,8 +81,8 @@ Step 6: Backup!
       :doc:`configuration page </appendix/backup-and-restore/configuration>`.
 
 Step 7: Transfer your backup files
-   You'll find the backup location within the ``config`` file on the backup
-   directory (`/opt/zammad/contrib/backup`). Make sure to adjust the backup configuration on the destination
+   You'll find the backup location within the ``config`` file in the backup
+   directory. Make sure to adjust the backup configuration on the destination
```

Comment:
Sorry but revert that. You removed the default location of the most current backup version. Bad idea.

Reply:
Hmm not sure I get your comment, That directory path was added by me, you then mentioned it wasn't a good idea to hardcode it. So I removed it. 
Now I should put it back ?

Reply:
Ugh no sorry. I've not been able to read here.